docs/resources/Soundboard.md

```diff
@@ -0,0 +1,90 @@
+# Soundboard Resource
+
+Users can play soundboard sounds in voice channels, triggering a Voice Channel Effect Send Gateway event for users connected to the voice channel. Bots cannot play soundboard sounds. Soundboard use is controlled by the `USE_SOUNDBOARD` and `USE_EXTERNAL_SOUNDS` [permissions](#DOCS_TOPICS_PERMISSIONS).
+
+There is a set of [default sounds](#DOCS_RESOURCES_SOUNDBOARD/list-soundboard-default-sounds) available to all users. Soundboard sounds can also be [created in a guild](#DOCS_RESOURCES_SOUNDBOARD/create-guild-soundboard-sound); users will be able to use the sounds in the guild, and Nitro subscribers can use them in all guilds.
+
+Soundboard sounds in a set of guilds can be retreived over the Gateway using [Request Soundboard Sounds](#DOCS_TOPICS_GATEWAY_EVENTS/request-soundboard-sounds).
+
+### Soundboard Sound Object
+
+###### Soundboard Sound Structure
+
+| Field         | Type                                            | Description                                                                                  |
+| ------------- | ----------------------------------------------- | -------------------------------------------------------------------------------------------- |
+| name          | string                                          | the name of this sound                                                                       |
+| sound_id      | snowflake                                       | the id of this sound                                                                         |
+| id?           | snowflake                                       | the id of this sound                                                                         |
```

Comment:
What's the difference between this and the `sound_id`?

Reply:
> Looks like `id` and `sound_id` are always the same, but `id` is optional

Reply:
Oh sorry mb, missed that part...